spec/features/search_page_spec.rb

```diff
@@ -69,48 +46,29 @@
   end
 
   scenario 'Match publisher query against available publishers', js: true do
-    first_dataset = DatasetBuilder.new
-                      .with_title('Data About Tonka Trucks')
-                      .with_publisher('Tonka Trucks')
-                      .build
-
-    second_dataset = DatasetBuilder.new
-                      .with_title('Data About Toby')
-                      .with_publisher('Toby Corp')
-                      .build
-
-    index(first_dataset, second_dataset)
+    dataset1 = build :dataset, title: 'Wanted'
```

Comment:
Using factory default texts should make it easier to concentrate on the purpose of the test and less on the amusing content.